Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -23,12 +23,18 @@ interface CodeActionCommand {
     arguments?: any[];
     edit?: TextEdit;
     uri?: string;
+    range?: Range;
 }
 
 interface GetCodeActionsResult {
     commands: CodeActionCommand[];
 }
 
+export interface CreateDeclDefnCommandArguments {
```

Comment:
Do you need a new object with two field, and a single instance of that in the list of arguments?  Couldn't you just add both of these as arguments?  That would seem to be a little bit simpler, overall.

Reply:
Will address during refactor of code actions.